src/site/content/en/blog/usb/index.md

```diff
@@ -0,0 +1,369 @@
+it to the Web**.
+
+Let's see what you could expect with the WebUSB API:
+
+1. Buy a USB device.
+2. Plug it into your computer.
+3. A notification appears right away, with the right website to go to for this
+   device.
+4. Simply click on it. Website is there and ready to use!
+5. Click to connect and a USB device chooser shows up in Chrome, where you can
+   pick your device.
+6. Tada!
+
+What would this procedure be like without the WebUSB API?
+
+- Read a box, label, or search on line and possibly end up on the wrong website.
+- Have to install a native application.
+- Is it supported on my operating system? Make sure you download the "right"
+  thing.
+- Scary OS prompts popup and warn you about installing drivers/applications from
+  the Internet.
+- Malfunctioning code harms the whole computer. The Web is built to [contain
+  malfunctioning websites].
+- Only use the USB device once? On the Web, the website is gone once you closed
+  tab. On a computer the code sticks around.
+
+## Before we start
+
+This article assumes you have some basic knowledge of how USB works. If not, I
+recommend reading [USB in a NutShell]. For background information about USB,
+check out the [official USB specifications].
+
+The [WebUSB API] is available in Chrome 61.
+
+### Available for Origin Trials
+
+In order to get as much feedback as possible from developers using the WebUSB
+API in the field, we've previously added this feature in Chrome 54 and Chrome
+57 as an [origin trial].
+
+The latest trial has successfully ended in September 2017.
+
+## Privacy and security
+
+### HTTPS only
+
+Because this experimental API is a powerful new feature added to the Web, it is
+made available only to [secure contexts]. This means you'll need to build with
+TLS in mind.
+
+### User gesture required
+
+As a security feature, getting access to connected USB devices with
+`navigator.usb.requestDevice` must be called via a user gesture
+like a touch or mouse click.
+
+### Feature Policy
+
+A feature policy is a mechanism that allows developers to selectively enable
+and disable various browser features and APIs. It can be defined via a HTTP
+header and/or an iframe "allow" attribute.
+
+You can define a feature that controls whether the usb attribute is exposed on
+the Navigator object, or in other words if you allow WebUSB.
+
+Below is an example of a header policy where WebUSB is not allowed:
+
+```http
+Feature-Policy: fullscreen "*"; usb "none"; payment "self" https://payment.example.com
+```
+
+Below is another example of a different container policy where USB is allowed:
+
+```html
+<iframe allowpaymentrequest allow="usb; fullscreen"></iframe>
+```
+
+## Let's start coding
+
+The WebUSB API relies heavily on JavaScript [Promises]. If you're not familiar
+with them, check out this great [Promises tutorial]. One more thing, `() => {}`
+are simply ECMAScript 2015 [Arrow functions] -- they have a shorter syntax
```

Comment:
```suggestion
are simply ECMAScript 2015 [Arrow functions]. They have a shorter syntax
```

Reply:
Do we still need to explain promises and arrow functions?

Reply:
As noted in https://github.com/GoogleChrome/web.dev/pull/4090/#discussion_r508911986, I've removed the explanation but kept links as developers interested in Bluetooth or USB may not necessarily be web developers.